packages/dbos-compiler/templates/dbos.create.liquid

```diff
@@ -0,0 +1,88 @@
+CREATE EXTENSION IF NOT EXISTS plv8;
```

Comment:
I love how we (ab)use Liquid to write PL/pgSQL. It works!

Reply:
we're using, not abusing!